hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/ha/EditLogTailer.java

```diff
@@ -413,6 +413,8 @@ private boolean tooLongSinceLastLoad() {
     return new MultipleNameNodeProxy<Void>() {
       @Override
       protected Void doWork() throws IOException {
+        LOG.info("Triggering log scrolling to the remote NameNode, " +
```

Comment:
replace `log scrolling` with `log rolling`?

Reply:
The log already exists on line `427`: `LOG.info("Triggering log roll on remote NameNode");`. 
Should we remove the original log?

Reply:
Thanks @virajjasani  for the comment and review.
Yes, I agree with @tomscut's opinion. There is no need to record logs repeatedly.